### DISCUSSION TITLE
Custom component inside Date picker

### DISCUSSION THREAD

**Discussion by Jonyberry** [mantinedev/mantine#2873] [Feature requests]
Hi,

I was wondering if it's possible to implement option for rendering custom component inside Date picker.
I'm currently trying to add "Today" button and I'm little bit helpless.

Something like this ...
<img width="325" alt="date_picker" src="https://user-images.githubusercontent.com/17932351/199755237-e1779b74-621a-4a78-8f22-25e02e434792.png">

Thanks!

**Comment by rtivital** [comment]
No currently it is not supported in DatePicker component. You can build your own component for that using Calendar and Popover

**Comment by malted** [comment]
Hi, why was I mentioned?

**Reply by STCollier** [reply]
Was wondering the same thing

**Reply by IroncladDev** [reply]
it was a spam bot

**Reply by Cosmic6811** [reply]
Spam bot eh
